lib/tasks/hyacinth/ci.rake

```diff
@@ -29,7 +29,7 @@ namespace :hyacinth do
     ENV['INCLUDE_HYACINTH_SPECS'] = includes.join(',')
     rspec_system_exit_failure_exception = nil
 
-    task_stack = ['hyacinth:rspec']
+    task_stack = ['webpacker:clobber', 'webpacker:compile', 'hyacinth:rspec']
```

Comment:
Does this make the ci run take longer when a developer runs it locally?

If it doesn't have an effect on the actual ci functionality (and if you just added it to verify that the webpack compile tasks runs), should it be added only as an ENV config option that we enable in a CI environment like GitHub Actions?

Reply:
Also, maybe I'm misunderstanding your intention here, but here's some info from something I just tested:

As you'll recall, our earlier commit 6e2f831d2c35c16aa62cee4fe5dd7c3fc756db85 had this error when we tried to deploy:
```
ERROR in ./app/javascript/packs/hyacinth_v1.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'babel-plugin-transform-react-remove-prop-types'
...
...
```

I checked out that commit locally and ran:
```
bundle exec rake webpacker:clobber webpacker:compile
```
and that completed without error.

But when I ran this...
```
rm -rf node_modules; yarn install; rm -rf public/packs public/assets; mkdir public/packs; mkdir public/assets; NODE_ENV=production bundle exec rake assets:precompile
```
...it failed and printed that `Error: Cannot find module 'babel-plugin-transform-react-remove-prop-types'` error.

Reply:
I'm not sure what you're saying? In my case, the webpack compile was clearly executed at some point - but the script wasn't running (and the error is not available to a headless browser that I could see; blank screenshot outputs). When the tests ran with chrome instead of headless, then I could see the console (until the test timed out). That's where I was seeing this error.

Reply:
> Does this make the ci run take longer when a developer runs it locally?
> 
> If it doesn't have an effect on the actual ci functionality (and if you just added it to verify that the webpack compile tasks runs), should it be added only as an ENV config option that we enable in a CI environment like GitHub Actions?

I added some logic to make it optional (presuming it runs), but we need to acculturate the team to doing a test environment clobber/recompile if feature specs appear not to be executing the entry point js.

Reply:
I was saying that if your intention behind the addition of `webpacker:clobber` and `webpacker:compile` was to detect webpack build errors like the deployment one that we saw before, it wasn't working.  But it sounds like you're saying that it is not, and that it's only intended to detect a type of post-build runtime error that can appear in the chrome browser console -- is that right?

Btw, it may be good to clarify this intention with a comment above the `task_stack = ['webpacker:clobber', 'webpacker:compile', 'hyacinth:rspec']` line, since this seems like the thing that someone could forget down the line.

Reply:
> I added some logic to make it optional (presuming it runs), but we need to acculturate the team to doing a test environment clobber/recompile if feature specs appear not to be executing the entry point js.

Thank you for that.  I just tried running the task locally and it adds an extra 25 seconds to every run, so it's nice making that optional.